cli/utils.js

```diff
@@ -199,7 +199,7 @@ module.exports = {
                             {
                                 "handle": [
                                     {
-                                        "body": "{err.status_code} | {err.status_text} (on {http.regexp.host.1})",
+                                        "body": `502 (on ${domain.domain})`,
```

Comment:
hmm, 502 (should be) handled by the handler above so this could probably just be "unhandled error" or smth

Reply:
Who can fix it? Nest admins or the user?

Reply:
User (most of the time)